build.gradle

```diff
@@ -17,3 +17,8 @@ gatling {
 repositories {
   mavenCentral()
 }
+
+task gatlingRecorder(type: JavaExec) {
```

Comment:
Would rather have this in the plugin (as already discussed in the PR)

Reply:
100% agree